pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -115,9 +115,9 @@ spec:
   post: |
     {{- jsonpath .JsonResult "{.metadata.name}" | trim | addTo "putcstorpoolcr.objectName" .TaskResult | noop -}}
     {{- jsonpath .JsonResult "{.metadata.uid}" | trim | addTo "putcstorpoolcr.objectUID" .TaskResult | noop -}}
-    {{- jsonpath .JsonResult "{.metadata.labels.kubernetes\\.io/hostname}" | trim | addTo "putcstorpoolcr.nodeName" .TaskResult | noop -}}
-  task: |-
-    {{- $diskDeviceIdList:= .Storagepool.diskDeviceIdList }}
+    {{- $nodeUidList := jsonpath .JsonResult "pkey=nodeUid,{.metadata.labels.kubernetes\\.io/hostname}={.metadata.uid} {.metadata.name};" | trim | default "" | splitList ";" -}}
```

Comment:
What are we trying to do here? I guess this logic is over engineered? Can you please come up with a simpler alternative?

Reply:
done